File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <stdio.h>
#include <iostream>

using namespace std;

int main(){
    int a,b,c;
    scanf("%d\n%d\n%d", &a,&b,&c);
    if(a == 60 && b == 60 && c == 60){
        cout << "Equilateral" << endl;
    }
    else if(a+b+c == 180 && ((a==b && b !=c)||(a==c && a != b)||(b==c && a !=b))){
        cout << "Isosceles" << endl;
    }
    else if(a+b+c == 180 && (a != b && b != c && c != a)){
        cout << "Scalene" << endl;
    }
    else{
        cout << "Error" << endl;
    }
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringBuilder sb = new StringBuilder();

        int a = Integer.parseInt(br.readLine());
        int b = Integer.parseInt(br.readLine());
        int c = Integer.parseInt(br.readLine());
        int sum = a + b + c;

        if (a == 60 && a == b && b == c) {
            sb.append("Equilateral");
        } else if ((sum == 180) && (a == b || b == c || c == a)) {
            sb.append("Isosceles");
        } else if ((sum == 180) && (a != b && b != c && c != a)) {
            sb.append("Scalene");
        } else if (sum != 180) {
            sb.append("Error");
        }

        bw.write(sb.toString());
        bw.flush();
    }
}